utils/fscryptctl/patches/001-do_not_build_docs.patch

```diff
@@ -0,0 +1,11 @@
+--- a/Makefile
++++ b/Makefile
+@@ -133,7 +133,7 @@ test-all:
+ 
+ # Installation, uninstallation, and cleanup targets
+ 
+-all:fscryptctl fscryptctl.1
++all:fscryptctl
```

Comment:
I think you do not need to keep maintaining this patch if you set `MAKE_FLAGS := fscryptctl` in the Makefile instead.